appium/options/transformers.py

```diff
@@ -0,0 +1,13 @@
+from datetime import timedelta
+from typing import Optional, Union
+
+
+class DurationTransformer:
```

Comment:
I don't see a need to have these methods under a class. They could be simple functions

Reply:
Fixed. Moved functions out of the class.